sonar-ballerina-plugin/build.gradle

```diff
@@ -0,0 +1,249 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.util.jar.JarInputStream
+
+plugins {
+    id 'java'
+    id 'java-library'
+    id 'checkstyle'
+    id "com.github.spotbugs"
+    id "de.undercouch.download"
+    id "com.github.johnrengelman.shadow"
+    id "io.spring.dependency-management"
+}
+
+group = "${group}"
+version = "${sonarPluginVersion}"
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    gradlePluginPortal()
+
+    maven {
+        url = 'https://repox.jfrog.io/repox/plugins.gradle.org/'
```

Comment:
@Xenowa Whats the requirement of this maven repository?

Reply:
This maven repository is not required, Was used previously as other SonarQube plugins had it.